composer.json

```diff
@@ -23,7 +23,8 @@
     "autoload": {
         "psr-4": {
             "RA\\": "src/",
-            "Test\\": "tests/src"
+            "Test\\": "tests/src",
+			"Models\\": "lib/models"
```

Comment:
I'd reuse the existing `RA` namespace within `src` - a new `Models` directory in there seems fitting

Reply:
I'll make the change.